content/coronavirus_landing_page.yml

```diff
@@ -110,6 +110,14 @@ content:
               url: /coronavirus/worker-support
               featured_link: true
               description: "Get coronavirus (COVID-19) support if you’re working less or not working - and find out how to stay safe at work."
+        - title: Popular content
+          list:            
+            - label: Redundancy: your rights 
```

Comment:
If the text includes a character that is special to YAML, it needs to be enclosed in quotes. `:` is one of those characters, so this line needs to be:
```yaml
            - label: "Redundancy: your rights"
```